trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf5340/config.cmake

```diff
@@ -9,3 +9,4 @@
 set(SECURE_UART1                        ON         CACHE BOOL      "Enable secure UART1")
 set(PSA_API_TEST_TARGET                 "nrf"      CACHE STRING    "PSA API test target")
 set(ITS_NUM_ASSETS                      "5"        CACHE STRING    "The maximum number of assets to be stored in the Internal Trusted Storage area")
+set(PS_CRYPTO_AEAD_ALG                  PSA_ALG_CCM CACHE STRING    "The AEAD algorithm to use for authenticated encryption in Protected Storage")
```

Comment:
Do we need to switch to CCM on 53?

Reply:
Good question. @Vge0rge ?

Reply:
I thought that we will use the same algorithm for both just to have to have the same scheme. We definitely don't "need" to do that for the 5340 since GCM is supported so I will just update the PR.